examples/wordpress/README.md

```diff
@@ -42,7 +42,7 @@ The template contains the following variables:
 
 ```
 {% set PROPERTIES = properties or {} %}
-{% set PROJECT = PROPERTIES['project'] or 'dm-k8s-prod' %}
+{% set PROJECT = PROPERTIES['project'] or 'dm-k8s-testing' %}
```

Comment:
Just want to make sure these are all intentional, and not left over? I think @jackgr changed all these?